doc-content/enterprise-only/admin-and-config/managing-business-central-using-settings-xml-ref.adoc

```diff
@@ -0,0 +1,18 @@
+[id='managing-business-central-using-settings-xml-ref']
+= Configuring Maven using settings.xml file
+
+Java application development uses the Apache Maven distributed build automation tool to build and manage software projects. Maven uses Project Object Model (POM) configuration XML files to define project properties and manage the build process.
```

Comment:
maven is not distributed

Reply:
` project properties and manage the build process` -> not manage the build, but define the build

Reply:
changed as suggested.